test/cri-containerd/sandbox.go

```diff
@@ -34,3 +34,15 @@ func removePodSandbox(t *testing.T, client runtime.RuntimeServiceClient, ctx con
 		t.Fatalf("failed RemovePodSandbox for sandbox: %s, request with: %v", podID, err)
 	}
 }
+
+func getRunPodSandboxRequest(t *testing.T, runtimeHandler string) runtime.RunPodSandboxRequest {
+	return runtime.RunPodSandboxRequest{
```

Comment:
I like this, might be useful to replace all of the easy (e.g. nothing other than the runtime handler needing to be filled out) RunPodSandboxRequests we have in our tests with this also. Doesn't need to be this PR though :P

Reply:
I'll file a separate PR for this 😄